direnvrc

```diff
@@ -111,6 +111,15 @@ _nix_argsum_suffix() {
   fi
 }
 
+if [[ ! $(declare -f orig_watch_file) ]]; then
+  eval orig_"$(declare -f watch_file)"
+fi
+
+watch_file() {
```

Comment:
I wonder if it could problems down the line if we start override this function. It reminds a bit of rails monkey-patching ruby. cc @zimbatm

Reply:
Is it possible to rename it to nix_direnv_watch_file ? I would prefer not to have to sift through watch_file implementations in the upstream bug reports.

Reply:
Works for me as well. I named it nix_direnv_watch_file now.